sdjournal/read.go

```diff
@@ -188,6 +190,83 @@ process:
 	return
 }
 
+func (r *JournalReader) FollowWithFields(until <-chan time.Time, fields map[string]struct{}, writer io.Writer) (err error) {
```

Comment:
This is kinda copied from the Follow() function. But I feel it's better to just pass a time, then create a channel inside the function. (e.g. `untilCh := time.After(until)`)
cc @jonboulle ?

No need for a change in this PR though.